Add link to CLA folder for maintainers

OmarShehata opened the pull request:
The linked Google Drive folder is not a publicly accessible link, but is visible to members of the Cesium team. It's useful to have this link handy in cases when you need to manually check a CLA, like if the bot fails: https://github.com/CesiumGS/cesium/pull/9147#issuecomment-691075404.

cesium-concierge commented:
Thanks for the pull request @OmarShehata!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/OmarShehata-patch-1/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

hpinkos commented:
@OmarShehata I don't think this is the right place for that.  Maybe put a README.md in https://github.com/CesiumGS/cesium/tree/master/Documentation/Contributors/CLAs?

OmarShehata commented:
Are people going to find it there? I'm not sure how you'd end up there, that folder isn't really linked to from anywhere else. Maybe the best place is in concierge's comment itself, so it could be:

> :x:  Missing CLA.
>   * Please send in a [Contributor License Agreement (CLA)]() and comment back here to let us know to check this!
>   * For maintainers, see a list of signed CLAs in [this Google Drive folder]().

Since that's often when people actually need it.

OmarShehata commented:
@hpinkos I've moved it to a README under the CLAs folder as you suggested, and added a note for concierge when the CLA check fails or a CLA is not found since that is when maintainers will need to manually check the Google drive link.

hpinkos commented on Documentation/Contributors/CLAs/README.md:
```diff
@@ -0,0 +1,7 @@
+# Contributor License Agreements (CLA)
+
+Before we can review a pull request, we require a signed Contributor License Agreement. See [Contributing Guide](CONTRIBUTING.md#contributor-license-agreement-cla).
```
this URL for CONTRIBUTING.md isn't correct

OmarShehata commented:
Thanks @hpinkos , should be ready.

cesium-concierge commented:
Thanks again for your contribution @OmarShehata! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

OmarShehata commented:
@hpinkos can you merge or let me know if you have any other feedback?
